test/utils/bidOnTrade.ts

```diff
@@ -0,0 +1,36 @@
+import { Signer, constants } from 'ethers'
```

Comment:
I'm not a fan of this module, I'll rewrite tomorrow before anyone wastes time reviewing it.